mh/constants.py

```diff
@@ -326,8 +326,8 @@ def slot(item, qty):
         "content": make_binary_type_time_events
     },
     0x03: {
-        "version": 1,
-        "content": lambda: make_binary_npc_greeters(is_jap=IS_JAP)
+        "version": lambda binary_loader: current_event_time_slot() + 1,
+        "content": lambda temporal_slot: lambda binary_loader: make_binary_npc_greeters(binary_loader, is_jap=IS_JAP, temporal_slot=temporal_slot)
```

Comment:
Please provide functions/callbacks instead of lambdas to deduplicate the code and prevent the risk of mistakes. Ditto for the others below.

Reply:
This has been done